contracts/optimistic-ethereum/mockOVM/accounts/mockOVM_ECDSAContractAccount.sol

```diff
@@ -58,14 +58,23 @@ contract mockOVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
 
             // EVM doesn't tell us whether a contract creation failed, even if it reverted during
             // initialization. Always return `true` for our success value here.
-            return (true, abi.encode(created));
+            return (created != address(0), abi.encode(created));
```

Comment:
I think we can get rid of this file with the EOA updates ( not sure what we needed this mock for in the first place)

Reply:
this is actually exclusively used in the JSVM, leaving for those purposes.